snap/hooks/prepare-plug-edgex-secretstore-token

```diff
@@ -1,48 +1,8 @@
 #!/bin/bash
 
-# This connection can happen before or after security-secretstore-setup
-# (a oneshot service that configures Vault) has already run. The first
-# case happens when the content interface is auto-connected and the
-# external snap is installed before the edgexfoundry
-# snap. In this case there's no action required as security-secretstore-
-# -setup will write it's token to the source dir which has been bind
-# mounted over the dir of the same name in this snap.
+# This script needs to exist, for an unknown reason to avoid the following error:
```

Comment:
@anonymouse64 - could you comment on why not including this empty script causes this error?

Reply:
sorry meant the other comment to be a reply, see this: https://github.com/edgexfoundry/edgex-go/pull/3826/files/3b0cf43c7e43d1d5d2a17e9468597ccdf90f6ff1#r759344916